### ISSUE TITLE
OSX Compatibility

### ISSUE THREAD

**Issue by abbic** [vshaxe/flash-debugger#23]
The usage instructions seem to only cover Windows or Linux installations. I tried to follow the Linux instructions as best i could but result with this error.
```running /Users/abbi/projects/scopa/bin/flash/bin/scopa.swf
Error: Command failed: "/Users/abbi/projects/scopa/bin/flash/bin/scopa.swf"
/bin/sh: /Users/abbi/projects/scopa/bin/flash/bin/scopa.swf: Permission denied
```
Trying all kinds of things like modifying permissions has not helped. the `java` command is in my path. .swf files are associated with the player, but `flashplayer` is not globally accessible due to OSX constraints.

Is OSX the issue here?

**Comment by Geokureli** [comment]
I'm also having this issue. It seems that you cannot run a swf directly in OSX. if you were to open terminal and run
```/Users/abbi/projects/scopa/bin/flash/bin/scopa.swf```
you would get the same error, where this would work:
```open /Users/abbi/projects/scopa/bin/flash/bin/scopa.swf```

**Comment by Gama11** [comment]
Should be fixed in 1.2.3.
